lib/lob/client.ex

```diff
@@ -11,7 +11,7 @@ defmodule Lob.Client do
 
   @client_version Mix.Project.config[:version]
 
-  @type response :: {:ok, map, list} | {:error, map}
```

Comment:
Looks like theres some [extra](https://travis-ci.org/lob/lob-elixir/jobs/487207305#L1444) type specs that still reference this:

```sh
$ grep -r "Client.response" .                                                                                                                                                                                     ~/lob/lob-elixir-clones/lob-elixir
./lib/lob/bank_account.ex:  @spec verify(String.t, map, map) :: Client.response
./lib/lob/us_verification.ex:  @spec verify(map, map) :: Client.response
./lib/lob/resource_base.ex:        @spec list(map, map) :: Client.response
./lib/lob/resource_base.ex:        @spec retrieve(String.t, map) :: Client.response
./lib/lob/resource_base.ex:        @spec create(map, map) :: Client.response
./lib/lob/resource_base.ex:        @spec delete(String.t, map) :: Client.response
./lib/lob/us_zip_lookup.ex:  @spec lookup(map, map) :: Client.response
./lib/lob/intl_verification.ex:  @spec verify(map, map) :: Client.response
./lib/lob/us_autocompletion.ex:  @spec autocomplete(map, map) :: Client.response
```